g-leaflet-shramov/pom.xml

```diff
@@ -23,7 +23,7 @@
         <dependency>
             <groupId>org.peimari</groupId>
             <artifactId>g-leaflet</artifactId>
-            <version>0.4.11</version>
+            <version>1.0.0-b2</version>
```

Comment:
There is already 1.0.0-b3, with your awesome improvements, maybe build against that?